### PR TITLE
Update es-index.html and corrections to EN, ES, FR for 3_8_0

### PR DESCRIPTION
**Pull request by ignotus666**
# Does this need translation?

- [ ] Yes 
- [x] No 

# Changes

### CODE REVIEW

**Comment by ignotus666** [comment]
I've added a bunch of corrections to this PR:

- Changed en.FAQ.md to en-FAQ (same for ES and FR).
- `en-Installation-for-Linux.md`, `en-Software-Manual.md` and `en-Setup.md` get updated to their latest version (no need to merge them from 'changes' any more).
- Small formatting changes to the EN docs so they behave properly with po4a.
- Correction of errors in ES docs.
- Correction of formatting errors and links (might not have caught them all but a few have been corrected) in FR docs.
- Added LED screenshots for FR.

There have been no changes to EN content save the files mentioned above. I built the site locally and everything works. If we make the transition to po4a this will make it smoother because when further changes (changing links, renaming files, etc.) are made from now until the release they will happen to compatible files, making updating my branch much easier. If we finally don't make the transition, nothing changes but these corrections make it in before the release anyway.

**Comment by jujudusud** [comment]
Since images doesn't have to be translates (no text), it should stay only in the en directory.
The images are not différents from language to language.

**Comment by ignotus666** [comment]
The problem I see with that is that then you have to keep remembering which screenshots should have the language prefix changed and which shouldn't. If you include them all in each language folder you can just change the prefixes in all links without getting confused.

**Comment by jujudusud** [comment]
Not a problem from my point of view because it is made just one time and at the same time as creating the screenshots.
The best solution for me is to let the non translated images in another place, not in en directory. Why not directly in screenshots ?
That way, only the translated images are copied and modified.

**Comment by ignotus666** [comment]
The thing is that creating exceptions to rules leads to more complexity. I'm looking at this from a translator's perspective. I'd prefer to be told "Whenever you see a screenshot link, change the prefix to your language" than "Change the prefix to your language except when, um, it's a screeshot that doesn't need to be translated" What? So while I'm translating I have to be constantly checking whether screenshots are translated or not?

I'm going to give you an example that you have provided yourself (please don't take this as criticism - it's not):

When translating the FR docs, you followed the "rule" whereby links to other documents have to be left unchanged. Ah! but it turns out there are some "special" links, a part of which _does_ need to be translated: those that refer to a specific paragraph with a #. You understandably hadn't translated any of those (I think most of them have been corrected in this PR), and I myself find them particularly annoying as you have to either remember exactly how you titled that paragraph or search for it to check. Thankfully now there are far fewer than before though.

It could also be argued that creating yet another folder for untranslated screenshots just adds to the clutter... my point is that I think it's preferable to reduce complexity for translators as much as possible because it all adds up - 4 extra untranslated screeenshots out of more than 20 for each language isn't going to break the Internet.

**Comment by jujudusud** [comment]
If the responsible of Jamulus then, it's ok for me ;-)
But I don't think it is a good Idea ether.

File: wiki/en/en-Software-Manual.md [hunk 1]
```diff
@@ -4,104 +4,94 @@ title: "Software Manual"
 lang: "en"
 permalink: "/wiki/Software-Manual"
 ---
-# Jamulus Software Manual
+# Jamulus User Manual
 
-## Main Window
+{:.no_toc}
```
Comment (ann0see) [inline wiki/en/en-Software-Manual.md:9]: What's this class?

File: wiki/es/es-Running-a-Server.md [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
+title: 'Ejecutar un Servidor'
 ---
 
 # Ejecutar un Servidor
 
 ## ¿Necesito ejecutar un servidor para utilizar Jamulus?
 
-NO.
-{: .doubletextsize .red }
+NO. {: .doubletextsize .red }
```
Comment (ann0see) [inline wiki/es/es-Running-a-Server.md]: Are you sure this works with jekyll?

File: wiki/es/es-Server-Rpi.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
+category: hidden
```
Comment (ann0see) [inline wiki/es/es-Server-Rpi.md]: You can and should delete this file since it only needs to exist once (for the english version)

File: wiki/es/es-Tips-Tricks-More.md [hunk 4]
```diff
@@ -120,25 +121,23 @@ Para enviar estas señales utilizando systemd, crea los siguientes dos archivos
...
 ~~~
 
 Para empezar una nueva grabación:
 
 ~~~
-[Unit]
-Description=Toggle recording state of Jamulus server
-Requisite=Jamulus-Server
+ [Unit]
+ Description=Start a new recording on Jamulus server
```
Comment (ann0see) [inline wiki/es/es-Tips-Tricks-More.md:136]: This is used for switching recording on and of. Toggle is the correct word here.

File: wiki/fr/fr-Central-Servers.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (ann0see) [inline wiki/fr/fr-Central-Servers.md]: Can be deleted (as all the other redirects)

File: wiki/fr/fr-Choosing-a-Server-Type.md [hunk 6]
```diff
@@ -40,18 +42,17 @@ Ce mode est activé en décochant la case "Rendre mon serveur public...".
...
 
-Pour afficher les serveurs répertoriés par un serveur d'annuaire personnalisé, les musiciens doivent saisir l'adresse dans le champ de paramètres "Adresse personnalisée du serveur annuaire" de leur client.  Ils verront alors une liste de configuration de connexion générée par ce serveur d'annuaire.
+Pour afficher les serveurs répertoriés par un serveur d'annuaire personnalisé, les musiciens doivent saisir l'adresse dans le champ de paramètres "Adresse personnalisée du serveur annuaire" de leur client. Ils verront alors une liste de configuration de connexion générée par ce serveur d'annuaire.
 
-Les opérateurs de serveurs normaux peuvent également s'inscrire auprès de votre serveur d'annuaire personnalisé afin que leurs serveurs apparaissent dans votre liste de serveurs en définissant le votre comme option `--centralserver`.
+Les opérateurs de serveurs normaux peuvent également s'inscrire auprès de votre serveur d'annuaire personnalisé afin que leurs serveurs apparaissent dans votre liste de serveurs en définissant le votre comme option `--centralserver`. i
```
Comment (ann0see) [inline wiki/fr/fr-Choosing-a-Server-Type.md]: What's the i for?

**Review by ann0see — CHANGES_REQUESTED** [review]
Some comments. Could you please check if it builds locally via jekyll?

File: wiki/en/en-Software-Manual.md (hunk 1 above)
Comment (ignotus666) [reply wiki/en/en-Software-Manual.md:9]: No idea what that is - it's in the EN doc.

Comment (ignotus666) [reply wiki/en/en-Software-Manual.md:9]: Oh and yes, it builds and displays correctly building it locally.

File: wiki/es/es-Running-a-Server.md (hunk 2 above)
Comment (ignotus666) [reply wiki/es/es-Running-a-Server.md]: Well spotted. It actually has to be:

NO.
 {: .doubletextsize .red }

With a space at the start of the second line for it to work properly with po4a.

File: wiki/es/es-Server-Rpi.md (hunk 3 above)
Comment (ignotus666) [reply wiki/es/es-Server-Rpi.md]: Done. The trouble is that all files in the EN directory will have their template and .po file created. But if we leave them untranslated they won't get coverted to .md. This one probably got translated (the same with the other redirects which I'll remove).

File: wiki/es/es-Tips-Tricks-More.md (hunk 4 above)
Comment (ignotus666) [reply wiki/es/es-Tips-Tricks-More.md:136]: If it's wrong it has to be changed in the original. This is merely a copy/paste from there.

File: wiki/fr/fr-Central-Servers.md (hunk 5 above)
Comment (ignotus666) [reply wiki/fr/fr-Central-Servers.md]: Will do.

File: wiki/es/es-Tips-Tricks-More.md (hunk 4 above)
Comment (ann0see) [reply wiki/es/es-Tips-Tricks-More.md:136]: Strange. @gilgongo might know something about this?

File: wiki/fr/fr-Choosing-a-Server-Type.md (hunk 6 above)
Comment (ignotus666) [reply wiki/fr/fr-Choosing-a-Server-Type.md]: Oops, must have slipped in due to a careless finger...

File: wiki/en/en-Running-a-Server.md [hunk 7]
```diff
@@ -10,7 +10,7 @@ permalink: "/wiki/Running-a-Server"
 ## Do I need to run a server to use Jamulus?
 
 NO.
-{: .doubletextsize .red }
+ {: .doubletextsize .red }
```
Comment (ann0see) [inline wiki/en/en-Running-a-Server.md:13]: Will check your pr locally.

File: wiki/es/es-Choosing-a-Server-Type.md [hunk 8]
```diff
@@ -12,52 +12,50 @@ permalink: "/wiki/Choosing-a-Server-Type"
...
 Tu servidor aparecerá en la lista del [servidor de directorio](Directory-Servers) que los clientes utilizan por defecto. Los músicos ya pueden ver y conectarse a tu servidor. No es necesario ejecutarlo como un servidor público ya que muchas otras personas ya los están ejecutando. Simplemente conéctate a uno con latencia baja y empieza a tocar.
 
-<figure><img src="{{site.url}}/assets/img/es-screenshots/diagram-public-server.png" loading="lazy" alt="Diagrama de conexiones entre clientes dentro de un servidor público de Jamulus"></figure>
+<figure><img src="{{site.url}}/assets/img/es-screenshots/diagram-public-server.png" loading="lazy" alt="Diagram of connections between clients within a Jamulus public server"></figure>
 
 Este modo se activa con la casilla de "Registrar mi servidor..." y (opcionalmente) rellenando los campos de información sobre tu servidor.
 
 Cuando se ejecuta sin GUI ("headless"), lo siguiente configuraría un servidor público:
 
-~~~
```
Comment (ann0see) [inline wiki/es/es-Choosing-a-Server-Type.md:23]: Why was this removed?

File: wiki/es/es-Command-Line-Options.md [hunk 9]
```diff
@@ -11,15 +11,11 @@ Las funciones más comunes de Jamulus pueden configurarse usando la interfaz gr
 
 Por ejemplo, en Windows, para que un cliente utilice un archivo específico de configuración, haz clic derecho en el acceso directo de Jamulus y elige "Propiedades" > Destino. Añade los argumentos necesarios a Jamulus.exe:
 
-```shell
```
Comment (ann0see) [inline wiki/es/es-Command-Line-Options.md:14]: Why was this removed?

File: wiki/fr/fr-Client-Troubleshooting.md [hunk 10]
```diff
@@ -79,9 +79,10 @@ Voir [cette discussion du forum](https://sourceforge.net/p/llcon/discussion/5335
 ### Vous utilisez un Mac et votre entrée n'est pas entendue ?
 
 (Avec les remerciements de [Mark Anthony De Souza](https://www.facebook.com/groups/619274602254947/permalink/765122847670121/?comment_id=765525034296569))
-Vous n'avez peut-être pas répondu "Oui" à la question "Jamulus veut accéder à votre microphone".  Pour résoudre ce problème :
+
+Vous n'avez peut-être pas répondu "Oui" à la question "Jamulus veut accéder à votre microphone". Pour résoudre ce problème :
 * Allez dans `Préférences` > `Sécurité et confidentialité` > onglet `Privacy`.
-* Trouvez "Microphone" sur la gauche et assurez-vous que "Jamulus" est activé sur la liste de droite.
+* Trouvez `Microphone` sur la gauche et assurez-vous que `Jamulus` est activé sur la liste de droite.
```
Comment (ann0see) [inline wiki/fr/fr-Client-Troubleshooting.md:85]: Why was this changed? \` is used for code/CLI commands usually.

File: wiki/es/es-Server-Rpi.md (hunk 3 above)
Comment (ann0see) [reply wiki/es/es-Server-Rpi.md]: Ah. Ok.

File: wiki/en/en-Running-a-Server.md (hunk 7 above)
Comment (ignotus666) [reply wiki/en/en-Running-a-Server.md:13]: Yes. It works.

Comment (ann0see) [reply wiki/en/en-Running-a-Server.md:13]: Ok. Thanks for the clarification here.

File: wiki/fr/fr-Client-Troubleshooting.md (hunk 10 above)
Comment (ignotus666) [reply wiki/fr/fr-Client-Troubleshooting.md:85]: You'd have to ask @jujudusud. I didn't change any of the FR content except obvious missing **, _  and links

**Comment by ann0see** [comment]
I think once my comments are resolved, you can merge this one. I think I'll have to go through the whole website since there seem to be a lot of work needed: broken links, missing images like the profile window; ... I hope the translation process is done so that I can fix all this stuff.

File: wiki/es/es-Choosing-a-Server-Type.md (hunk 8 above)
Comment (ignotus666) [reply wiki/es/es-Choosing-a-Server-Type.md:23]: Fixed

File: wiki/es/es-Command-Line-Options.md (hunk 9 above)
Comment (ignotus666) [reply wiki/es/es-Command-Line-Options.md:14]: Fixed

**Comment by ann0see** [comment]
I'll merge this one now and will hunt broken stuff on the website.

File: wiki/fr/fr-Client-Troubleshooting.md (hunk 10 above)
Comment (ann0see) [reply wiki/fr/fr-Client-Troubleshooting.md:85]: Ok. Thanks for this informaiton. Seems as if I need to invest some more time in the website before we can release it.

Comment (ignotus666) [reply wiki/fr/fr-Client-Troubleshooting.md:85]: Actually, it's like that in the original EN document - @jujudusud simply followed the same style.

**Comment by ignotus666** [comment]
Ok thanks. If you do your fixes on the files as they are now it'll make life much easier for me to keep my po4a branch in sync.

File: wiki/fr/fr-Client-Troubleshooting.md (hunk 10 above)
Comment (ann0see) [reply wiki/fr/fr-Client-Troubleshooting.md:85]: Ok. Will open a PR to the main repo shortly...

File: wiki/es/es-Tips-Tricks-More.md (hunk 4 above)
Comment (gilgongo) [reply wiki/es/es-Tips-Tricks-More.md:136]: As far as I know there is a distinction between toggling recordings on and off using SIGUSR2 and starting a new one using SIGUSR1. The latter creates a new directory to save the file to, and you use the former to control whether it's saving or not. So the docs are correct I think. I guess if there's a confusion between "on/off" and "new" we could clarify that maybe (eg "To control whether the recording is running or not")?

Comment (gilgongo) [reply wiki/es/es-Tips-Tricks-More.md:136]: BTW once 3.8 is out I plan to create a "Server Administration Guide" that will clarify this kind of stuff as it's currently a bit scattered between separate files (same for "User Manual").

File: wiki/fr/fr-Client-Troubleshooting.md (hunk 10 above)
Comment (jujudusud) [reply wiki/fr/fr-Client-Troubleshooting.md:85]: Yes, same as EN for me. only translating ... ;-)
